### ISSUE TITLE
UI New Repository/Migration : show full organisation name in menu selection

### ISSUE THREAD

**Issue by tst2005** [go-gitea/gitea#4318]
- Gitea version (or commit ref): 1.4.2
- Git version: 2.1.4
- Operating system: Linux Debian 64bits
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [x] Yes (It should but try.gitea.io is currently down)
  - [ ] No
  - [ ] Not relevant
- Log gist:

## Description

It should be good to see full organisation names to choose the good one.
in "New repository" or "New Migration".

I fail to work with organisation like : 
* tst2005-github-pull-readonly
* tst2005-github-push-done
* tst2005-github-push-todo 
* etc.

## Screenshots
![gitea-ui-show-org-long-name](https://user-images.githubusercontent.com/1117821/41901916-759952a4-7932-11e8-8bf2-e271bbb8d46c.png)

**Comment by tst2005** [comment]
The name is also truncated on the page refreshed just after an organisation created.

![gitea-ui-create-org-name-truncated](https://user-images.githubusercontent.com/1117821/41975765-ee892074-7a1b-11e8-813e-2e86513ff1f3.png)

The top rigth button truncate the name at 7 characters ... really ?!

**Comment by tst2005** [comment]
*recent activity*
@stale bot please just go die ;-)

**Comment by 6543** [comment]
@tst2005 still needed?

mouse houve print help text with full name

**Comment by tst2005** [comment]
@6543 yes I still need it. it is still not critical, just QoL. Thanks!

**Comment by zeripath** [comment]
OK basically the issue is that we use `.ShortName 20` to deliberately shorten the name to 20 characters.

I've had a look at trying to add `class="text truncate"` in opportune places but we would need to adjust the y-size of the inline blocks to make sure that the icons aren't clipped and that descenders and ascenders in the text aren't simply clipped.

It's a bit fiddly.

**Comment by tst2005** [comment]
I understand. The UI world is a maze of pain :P
